cms/server/admin/handlers/student.py

```
@@ -541,422 +464,31 @@ def post(self, training_program_id: str, user_id: str):
             tags_str = self.get_argument("student_tags", "")
-            student.student_tags = parse_tags(tags_str)
+            self.student.student_tags = parse_tags(tags_str)
 
             if self.try_commit():
-                self.write({"success": True, "tags": student.student_tags})
+                self.write({"success": True, "tags": self.student.student_tags})
             else:
                 self.set_status(500)
                 self.write({"error": "Failed to save"})
 
         except Exception as error:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)